### PR TITLE
Closes #160 - Adds `People you might know` components

### LINKED ISSUE #160: Create New BB: People You Might Know Section
This is a Building Block for our upcoming Social Media Kit. For anyone who wants to contribute some building blocks to the site, feel free to comment below to let us know you're taking this one on!

**Description/Interaction:** This could be a side bar or it's own section below some statuses... or something else! Feel free to vary from the sketch. Should include at least: header, profile pic & name, some kind of action button (add friend, follow, etc..). 

Please use category "containers" in yml file. Have fun!

**Example:**
screen shot 2017-04-25 at 10 25 34 am

### PR DESCRIPTION
**Pull request by IamManchanda**
Closes #160

PS: Anything **common** in these below demo profiles. Yup birthday on **4th of august**!

people-you-might-know

### CODE REVIEW

**Comment by rafibomb** [comment]
@IamManchanda haha, you're in good company then!

File: src/building-blocks/people-you-might-know/people-you-might-know.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<div class="people-you-might-know">
+  <div class="add-people-header">
+    <h6 class="header-title">
+      People You Might Know
+    </h6>
+  </div>
+  <div class="row add-people-section">
```
Comment (canterberrie) [inline src/building-blocks/people-you-might-know/people-you-might-know.html]: Hmm.. not sure about sharing .row with .add-people-section. @rafibomb thoughts?

File: src/building-blocks/people-you-might-know/people-you-might-know.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    <div class="small-12 medium-6 columns about-people">
```
Comment (canterberrie) [inline src/building-blocks/people-you-might-know/people-you-might-know.html]: also not sure about sharing .column with .about-people. I usually like to separate columns from component-specific containers, but if you have a reason for doing it this way, I could be convinced of it!

**Comment by IamManchanda** [comment]
@canterberrie Code brevity is the reason!  
The code was getting too longer
I think this way it's good !

There is no such manual that row and columns classes should have other classes!

**Comment by IamManchanda** [comment]
Moreover it was requiring lot of nesting.. I think this way its also ohk

**Comment by canterberrie** [comment]
@IamManchanda Sounds good to me! Merging :)

**Pull request merged**